lib/injekkt.rb

```diff
@@ -0,0 +1,30 @@
+class Array
+
+  def injekkt(*args, &block)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for injekkt is too high. [19.62/15]